nexuse2e-webapp/src/main/webapp/WEB-INF/pages/reporting/statistics.jsp

```diff
@@ -24,57 +24,262 @@
-	No messages have been created within the last 24 hours. Please try again when messages have been received or sent.
-</logic:lessEqual>
-
-<center><logic:messagesPresent>
+			<logic:iterate id="choreography" name="choreographies">
+				<tr>
+					<td title="${choreography.name}"><nexus:link
+							href="ChoreographyView.do?nxChoreographyId=${choreography.nxChoreographyId}"
+							styleClass="NexusLink">
+						${choreography.name}
+					</nexus:link></td>
+					<td title="${choreography.lastInboundTime}">
+						${choreography.lastInboundTime}
+					</td>
+					<td title="${choreography.lastOutboundTime}">
+						${choreography.lastOutboundTime}
+					</td>
+				</tr>
+			</logic:iterate>
+			</c:when>
+			<c:otherwise>
+				<tr>
+					<td class="no-data">no choreographies</td>
+				</tr>
+			</c:otherwise>
+		</c:choose>
+	</table>
+
+	<table class="NEXUS_TABLE fixed-table tabcontent" id="certificates">
+		<c:choose>
+			<c:when test="${atLeastOneCertificateConfigured}">
+				<tr>
+					<th class="NEXUSSection">Configured For</th>
+					<th class="NEXUSSection">Certificate Name</th>
+					<th class="NEXUSSection">Time Remaining</th>
+				</tr>
+				<logic:iterate id="cert" name="localCertificates">
+					<tr class="local">
+						<td class="NEXUSName" title="Local">Local</td>
+						<td class="NEXUSName" title="${cert.name}">
+							<nexus:link styleClass="NexusLink"
+										href="StagingCertificateView.do?nxCertificateId=${cert.nxCertificateId}">
+								${cert.name}
+							</nexus:link>
+						</td>
+						<td class="NEXUSName" title="${cert.remaining}">${cert.remaining}</td>
+					</tr>
+				</logic:iterate>
+				<logic:iterate id="partner" name="partners">
+					<c:forEach var = "cert" items="${partner.certificates}">
+						<tr>
+							<td title="${partner.name}">${partner.name}</td>
+							<td title="${cert.name}">
+								<nexus:link styleClass="NexusLink"
+											href="PartnerCertificateView.do?nxPartnerId=${partner.nxPartnerId}&nxCertificateId=${cert.nxCertificateId}">
+									${cert.name}
+								</nexus:link>
+							</td>
+							<td title="${cert.remaining}">${cert.remaining}</td>
+						</tr>
+					</c:forEach>
+				</logic:iterate>
+			</c:when>
+			<c:otherwise>
+				<tr>
+					<td class="no-data">no certificates configured or staged</td>
+				</tr>
+			</c:otherwise>
+		</c:choose>
+	</table>
+</div>
+
+
+<script>
+function openTab(evt, contentId) {
+	var tabContent = document.querySelectorAll(".tabcontent");
+	for (var i = 0; i < tabContent.length; i++) {
+		tabContent[i].style.display = "none";
+	}
+
+	var tabLinks = document.querySelectorAll(".tablinks");
+	for (var i = 0; i < tabLinks.length; i++) {
+		tabLinks[i].className = tabLinks[i].className.replace(" active", "");
+	}
+
+	document.getElementById(contentId).style.display = "table";
+	var target = (evt.currentTarget) ? evt.currentTarget : evt.srcElement;
+	target.className += " active";
+}
+
+document.querySelector('.tablinks').click();
```

Comment:
I do not understand what this is for. Maybe a comment to explain the background.